google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/debugger/BreakpointUtil.java

```diff
@@ -84,11 +86,17 @@ public static Date parseDateTime(@Nullable String dateString) {
     dateString = dateString.replaceAll("Z$", "-0000");
 
     SimpleDateFormat iso8601Format = new SimpleDateFormat(ISO_8601_FORMAT);
+    iso8601Format.setLenient(true);
 
     try {
```

Comment:
Agree I guess. I just don't really like the nested try catch but it's not a big deal.